tests/50federation/31room-send.pl

```diff
@@ -145,3 +146,58 @@
          });
       });
    };
+
+# Test for MSC2228 for messages received through federation.
+test "Ephemeral messages received from servers are correctly expired",
+   requires => [ local_user_and_room_fixtures(), remote_user_fixture() ],
+
+   do => sub {
+      my ( $local_user, $room_id, $federated_user ) = @_;
+
+      my $filter = '{"types":["m.room.message"]}';
+
+      matrix_invite_user_to_room( $local_user, $federated_user, $room_id )->then( sub {
+         matrix_join_room( $federated_user, $room_id )
+      })->then( sub {
+         my $now_ms = int( time() * 1000 );
+
+         matrix_send_room_message( $local_user, $room_id,
+            content => {
+               msgtype                          => "m.text",
+               body                             => "This is a message",
+               "org.matrix.self_destruct_after" => $now_ms + 500,
```

Comment:
I'm surprised that the expiry time for the federated test is lower than that for the local test, and it seems to be different to what you wrote previously.

Reply:
Yeah, I've tried tweaking this one a bit to see if I could lower it. I'll probably try a few other values later (and harmonise them between both tests).